actors/builtin/miner/monies.go

```diff
@@ -211,15 +211,25 @@ func LockedRewardFromReward(reward abi.TokenAmount) (abi.TokenAmount, *VestSpec)
 }
 
 var EstimatedSingleProofGasUsage = big.NewInt(65733297) // PARAM_SPEC
-var BatchDiscount = builtin.BigFrac{                    // PARAM_SPEC
+var EstimatedSingleProveCommitGasUsage = big.NewInt(49299973)
```

Comment:
1) You should be able to nuke EstimatedSingleProofGasUsage above^
2) Out of curiosity, where do these numbers come from?

Reply:
They are rough estimates of the mean gas used per single PreCommit and ProveCommit.  Empirically measured from tests and validated with small samples of on chain gas values.  Wouldn't be surprised if they need tweaking down the line.